### PR TITLE
Fix favicon and other related metadata

### PR DESCRIPTION
**Pull request by rightsaidjames**
Fixed some bad assumptions made when customising the default template!

- Favicon now works
- Added `apple-touch-icon`
- Tweaked heading structure of `index.md` and added `title` property to metadata

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -1,33 +1,38 @@
...
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta name="theme-color" content="#157878">
-    <link rel="icon" href="/_includes/favicon.png">
-    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
-    <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
-    {% include head-custom.html %}
-  </head>
-  <body>
-    <a id="skip-to-content" href="#content">Skip to the content.</a>
+<head>
```
Comment (rightsaidjames) [inline _layouts/default.html:4]: The only thing that's actually changed here is that I took out the favicon link element.

File: _layouts/default.html [hunk 2]
```diff
@@ -1,33 +1,38 @@
...
-      <h2 class="project-tagline">{{ page.description | default: site.description | default: site.github.project_tagline }}</h2>
-        <a href="https://join.slack.com/t/indietestingcommunity/shared_invite/zt-1t73fz3y9-GxgExSyEtsXeiwzn5v_apQ" class="btn">Join the Slack</a>
-    </header>
+<body>
+  <a id="skip-to-content" href="#content">Skip to the content.</a>
 
-    <main id="content" class="main-content" role="main">
-      {{ content }}
+  <header class="page-header" role="banner">
+    <h1 class="project-name">{{ page.title }}</h1>
```
Comment (rightsaidjames) [inline _layouts/default.html:21]: I put this variable back

File: index.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
 ---
 layout: default
+title: Indie Testing Community
```
Comment (rightsaidjames) [inline index.md:3]: This is what populates the header H1 and the `page.title` variable.

File: index.md [hunk 4]
```diff
@@ -1,8 +1,9 @@
...
 ---
 
-# Welcome to the Indie Testing Community
+## Welcome to the Indie Testing Community
```
Comment (rightsaidjames) [inline index.md:6]: Downgraded this to a H2

**Review by rightsaidjames — COMMENTED** [review]
Just some code annotations!

File: _includes/head-custom.html [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{% include head-custom-google-analytics.html %}
+
+<link rel="icon" href="/images/itc-icon.png">
+<link rel="apple-touch-icon" href="/images/itc-icon.png">
```
Comment (will-banked) [inline _includes/head-custom.html:4]: Didn't know about this, good spot

**Review by will-banked — APPROVED** [review]

**Pull request merged**